config/routes.rb

```diff
@@ -1,3 +1,17 @@
 Rails.application.routes.draw do
+  resources :posts
```

Comment:
This statement should automate route creation for everything under lines 4-8 in this file. Try it out. Delete lines 4-8 and try to use your app to see what happens. Then, do some research as to what this statement actually does.

[`OPTIONAL`]

Reply:
Changed and working :+1: